src/applications/managing-app-configuration.md

```diff
@@ -80,12 +80,14 @@ zero is considered a pass, while a non-zero status code is considered a fail.
 healthy if the check can establish a connection. `tcpSocket` probes accept a port number to perform
 the socket connection on the Container.
 
-Health checks can be configured on a per-application basis using `deis healthchecks:set`. To
+Health checks can be configured on a per-proctype basis for each application using `deis healthchecks:set`. If no type is mentioned then the health checks are applied to all routable proctypes. To
```

Comment:
How is the user able to determine which process types are routable, especially now that we got `deis routing` in https://github.com/deis/workflow/pull/438?